tests/gpu/Lut3DOp_test.cpp

```diff
@@ -233,31 +120,78 @@ OCIO_ADD_GPU_TEST(Lut3DOp, arbitrary_using_CSP_file)
 
 
 
-#ifndef OCIO_UNIT_TEST_FILES_DIR
-#error Expecting OCIO_UNIT_TEST_FILES_DIR to be defined for tests. Check relevant CMakeLists.txt
-#endif
 
+OCIO_ADD_GPU_TEST(Lut3DOp, red_only_using_CSP_file_legacy_shader)
+{
+    // Any other 3D LUT file format would have been good also.
+    // Create the transform & set the unit test
 
-// For explanation, refer to https://gcc.gnu.org/onlinedocs/cpp/Stringizing.html
-#define _STR(x) #x
-#define STR(x) _STR(x)
+    OCIO::FileTransformRcPtr file = GetFileTransform("Lut3DOp_red_only_using_CSP_file_legacy_shader.csp");
+    file->setInterpolation(OCIO::INTERP_LINEAR);
 
-static const std::string ocioTestFilesDir(STR(OCIO_UNIT_TEST_FILES_DIR));
+    test.setProcessor(file);
 
+    test.setLegacyShader(true);
+    test.setErrorThreshold(2e-4f);
+}
 
-namespace
+OCIO_ADD_GPU_TEST(Lut3DOp, green_only_using_CSP_file_legacy_shader)

```

Comment:
Thanks for the updates Ananth!  However, it looks like you have a duplicate copy of the tests below that use the earlier LUT names. Please remove the duplicate tests.

Ideally, you should run the unit tests locally before pushing updates.   :)

Reply:
I knew that would come back to bite me. Done.